core/src/main/scala/akka/persistence/cassandra/journal/CassandraJournal.scala

```diff
@@ -800,9 +800,8 @@ import akka.stream.scaladsl.Source
           Extractors.optionalTaggedPersistentRepr(eventDeserializer, serialization))
         .mapAsync(1) { t =>
           t.tagged match {
-            case OptionVal.Some(tpr) =>
-              tr.sendMissingTagWrite(tp)(tpr)
             case OptionVal.None => FutureDone // no tags, skip
+            case tpr            => tr.sendMissingTagWrite(tp)(tpr.x)
```

Comment:
Why not just change to :
```scala
case OptionVal.Some(tpr) =>
              tr.sendMissingTagWrite(tp)(tpr.x)
```
?

Reply:
That would still cause the pattern match warning.